app/metal-controller-manager/main.go

```diff
@@ -193,6 +195,17 @@ func main() {
 		}
 	}()
 
+	k8sClient, err := client.NewClient(nil)
+	if err != nil {
+		setupLog.Error(err, `failed to create k8s client`)
+		os.Exit(1)
+	}
+
+	if err = controllers.ReconcileServerClassAny(context.TODO(), k8sClient); err != nil {
+		setupLog.Error(err, `failed to reconcile ServerClass "any"`)
+		os.Exit(1)
+	}
```

Comment:
We have to use a separate client there due to controller-runtime's caching "subsystem" is not running yet.

… Actually, I should use msg.GetAPIReader() I just found instead :(

Reply:
one more idea to handle this is to inject reconcilation for `any` at the beginning of the controller work

Reply:
I don't have any _awesome_ idea about it, but something in `SetupWithManager` should do the trick imho

Reply:
I actually tried to do that in `SetupWithManager` – cache is still not running at that point